src/test-gui/java/de/qabel/desktop/ui/accounting/item/AccountingItemGuiTest.java

```diff
@@ -35,9 +35,13 @@ protected FXMLView getView() {
     }
 
     @Test
+    public void testMenu() throws Exception {
+        page.openMenu();
```

Comment:
this test doesn't assert anything. It just clicks on the menu icon and succeeds, no matter if the click results in any reaction or nothing happens.